lib/solr/configuration.rb

```diff
@@ -1,16 +1,34 @@
+require 'solr/field'
+require 'solr/dynamic_field_type'
```

Comment:
can you please nest it inside `solr/configuration`? Otherwise we'll end up with a lot of top-level classes and modules